.github/workflows/openmetadata-ingestion-core-version-comment.yml

```diff
@@ -15,7 +15,7 @@ name: OpenMetadata Ingestion Core Version Check Comment
 # access to secrets
 on:
   pull_request_target:
-    branches: [ main ]
+    branches: [ main, '0.[0-9]+.[0-9]+' ]
```

Comment:
One quick enhancement suggestion here, let's follow a standard yaml pattern with below snippet everywhere in CI files!
```
branches:
      - main
      - '0.[0-9]+.[0-9]+'
```